Jenkinsfile

```diff
@@ -43,9 +49,20 @@ pipeline {
     }
     post {
         always {
-            archiveArtifacts artifacts: 'build/libs/**/*.jar', fingerprint: true
+            archiveArtifacts artifacts: 'build/libs/**/*.jar', fingerprint: true, onlyIfSuccessful: true
```

Comment:
this can be deleted, also we need the hook to cause the promote so the web page rebuilds.

Reply:
Wasn't sure if you meant deleting the whole line or just the onlyIfSuccessful part. I'm assuming the whole line.
Added hook for web page rebuilds.